src/storagetm/storagetm.c

```diff
@@ -1136,6 +1137,12 @@ static int run(int argc, char* argv[]) {
         if (r < 0)
                 return log_error_errno(r, "Failed to install exit signal handlers: %m");
 
+        r = path_is_mount_point("/sys/kernel/config");
```

Comment:
Could you add a short comment that the check is typically not necessary as the unit has dependency to the mount, and this is mostly for when the daemon is run outside of the service ?

Reply:
@DaanDeMeyer I mean, please add a short comment _in the code_.